package/Dockerfile

```diff
@@ -66,7 +66,9 @@ RUN curl -sLf ${!TINI_URL} > /usr/bin/tini && \
     curl -sfL ${!ETCD_URL} | tar xvzf - --strip-components=1 -C /usr/bin/ etcd-${CATTLE_ETCD_VERSION}-linux-${ARCH}/etcd etcd-${CATTLE_ETCD_VERSION}-linux-${ARCH}/etcdctl && \
     curl -sLf https://github.com/rancher/telemetry/releases/download/${TELEMETRY_VERSION}/telemetry-${ARCH} > /usr/bin/telemetry && \
     curl -sLf https://storage.googleapis.com/kubernetes-release/release/${KUBECTL_VERSION}/bin/linux/${ARCH}/kubectl > /usr/bin/kubectl && \
-    chmod +x /usr/bin/rancher-helm /usr/bin/tini /usr/bin/telemetry /usr/bin/rancher-tiller /usr/bin/k3s /usr/bin/kubectl
+    chmod +x /usr/bin/rancher-helm /usr/bin/tini /usr/bin/telemetry /usr/bin/rancher-tiller /usr/bin/k3s /usr/bin/kubectl && \
+    mkdir -p /var/lib/rancher/management-state/driver-metadata && \
+    curl -sLf https://releases.rancher.com/kontainer-driver-metadata/${RANCHER_METADATA_BRANCH}/data.json > /var/lib/rancher/management-state/driver-metadata/data.json
```

Comment:
`/var/lib/rancher` is what we document as mountable location from the host, please use the directory as shown in https://github.com/rancher/rancher/pull/25388/files#diff-fa24c4fe8d6b6653fad727c37ac24b99R33

Reply:
ah, ok. The file was historically stored in there but I will move it.

Reply:
@superseb updated